pydicom/tests/test_dataset.py

```diff
@@ -166,17 +169,17 @@ def testMembership(self):
         self.assertTrue(
             'Dummyname' not in ds, "non-member tested as member")
 
-    def testContains(self):
+    def test_contains(self):
         """Dataset: can test if item present by 'if <tag> in dataset'......."""
         ds = self.dummy_dataset()
         ds.CommandGroupLength = 100  # (0000,0000)
-        self.assertTrue((0x300a, 0xb2) in ds, "membership test failed")
-        self.assertTrue([0x300a, 0xb2] in ds,
-                        "membership test failed when list used")
-        self.assertTrue(0x300a00b2 in ds, "membership test failed")
-        self.assertTrue(
-            (0x10, 0x5f) not in ds, "non-member tested as member")
-        self.assertTrue('CommandGroupLength' in ds)
+        assert (0x300a, 0xb2) in ds
+        assert [0x300a, 0xb2] in ds
+        assert 0x300a00b2 in ds
+        assert (0x10, 0x5f) not in ds
+        assert 'CommandGroupLength' in ds
+        # Use a negative tag to cause an exception
```

Comment:
Is it really using an error because this test is just checking if the tag is in ds?